include/ogdf/simultaneous/SimDraw.h

```diff
@@ -154,7 +154,7 @@ class OGDF_EXPORT SimDraw {
 	void writeGML(const char* fileName) const;
 
 	//! returns graph consisting of all edges and nodes from SubGraph \p i
-	const Graph getBasicGraph(int i) const;
+	std::unique_ptr<GraphCopy> getBasicGraph(int i) const;
```

Comment:
We might want to mention this change in the porting guide as the function is public.

Reply:
Done!